lib/seadsa/ShadowMem.cc

```diff
@@ -1177,6 +1189,59 @@ void ShadowMemImpl::visitMemhavoc(CallSite &CS) {
   associateConcretePtr(memUse, ptr, &callInst);
 }
 
+void ShadowMemImpl::visitIsModified(CallSite &CS) {
+  LOG("dsa.ismodified", errs() << "Visiting isModified \n");
+  auto &callInst = *CS.getInstruction();
+  /* isModified definition:
+     bool @sea_is_modified(i8*)
+     operand 0 is the pointer to be marked
+  */
+  auto &ptr = *CS.getArgOperand(0);
+  if (!m_graph->hasCell(ptr)) {
```

Comment:
there might be a bitcast that needs to be removed. But you will see it in the first test.

Reply:
This seems to work as is in unit tests and verify-c-common also.